Gemfile

```diff
@@ -39,7 +41,9 @@ group :development do
   gem 'stackprof' # ruby 2.1+ only
 end
 
-gem 'data_services_api', git: 'https://github.com/epimorphics/ds-api-ruby.git'
+# rubocop:disable Layout/LineLength
```

Comment:
I don't care about Rubocop's opinion of the `Gemfile`, so I think rather than toggle Cops on and off in this file, we should just add an `Exclude` line to `.rubocop.yml`